Documents/Users/FeatureList.md

```diff
@@ -43,6 +43,11 @@ you can specify following line in .xvimrc.
 
      inoremap <C-e> <C-o>$
 
+## Navigation
+C-], C-t
+
+C-] is mapped to "Jump To Definition".
```

Comment:
https://github.com/neybis/XVim/blob/c078ed6aa45173df6ed3c45b31da57e4dff63204/XVim/XVimNormalEvaluator.m#L458-L461

It was actually already implemented and just not documented. When I added the documentation for C-t I also added the documentation for C-]

:)